docs/get-started/getting-started.md

```diff
@@ -375,9 +375,97 @@ and `GpuColumnarExchange`.  Those correspond to operations that run on the GPU.
+
+- **Transport**: Handles block transfers between executors using various means like: _NVLink_, _PCIe_, _Infiniband (IB)_, 
+_RDMA over Converged Ethernet (RoCE)_ or _TCP_, and as configured in UCX, in these scenarios:
+  - _GPU-to-GPU_: Shuffle blocks that were able to fit in GPU memory.
+  - _Host-to-GPU_ and _Disk-to-GPU_: Shuffle blocks that spilled to host (or disk), but will be manifested 
+  in the GPU in the downstream Spark task.
+
+In order to enable the _RapidsShuffleManager_, please follow these steps (If you don't have 
+Mellanox hardware go to *step 2*):
+
+1) If you have Mellanox NICs and an Infiniband(IB) or RoCE network, please ensure you have the 
+MLNX_OFED [driver installed](https://www.mellanox.com/products/infiniband-drivers/linux/mlnx_ofed), 
+and the [`nv_peer_mem` kernel module](https://www.mellanox.com/products/GPUDirect-RDMA).
+
+With `nv_peer_mem`, IB/RoCE-based transfers can perform zero-copy transfers directly from GPU memory.
+
+2) Install [UCX 1.8.1](https://github.com/openucx/ucx/releases/tag/v1.8.1). 
+
+3) You will need to configure your spark job with extra settings for UCX (we are looking to 
+simplify these settings in the near future):
+
+```shell
+...
+--conf spark.shuffle.manager=com.nvidia.spark.RapidsShuffleManager \
+--conf spark.shuffle.service.enabled=false \
+--conf spark.rapids.shuffle.transport.enabled=true \
+--conf spark.executorEnv.UCX_TLS=cuda_copy,cuda_ipc,rc,tcp \
+--conf spark.executorEnv.UCX_ERROR_SIGNALS= \
+--conf spark.executorEnv.UCX_MAX_RNDV_RAILS=1 \
+--conf spark.executorEnv.UCX_MEMTYPE_CACHE=n \
+--conf spark.executorEnv.UCX_RNDV_SCHEME=put_zcopy \
+--conf spark.executor.extraClassPath=/usr/lib:/usr/lib/ucx:${SPARK_CUDF_JAR}:${SPARK_RAPIDS_PLUGIN_JAR} \
+```
+
+Please note `extraClassPath`, presently requires the UCX libraries to be added to the classpath. Newer
+versions of UCX handle loading shared libraries differently, and should not require this.
+
+### UCX Environment Variables
+- `UCX_TLS`: 
+  - `cuda_copy`, and `cuda_ipc`: enables handling of CUDA memory in UCX, both for copy-based transport
+    and peer-to-peer communication between GPUs (NVLink/PCIe).
+  - `rc`: enables Infiniband and RoCE based transport in UCX.
+  - `tcp`: allows for TCP communication in cases where UCX deems necessary.
+- `UCX_ERROR_SIGNALS=`: Disables UCX signal catching, as it can cause issues with the JVM.
+- `UCX_MAX_RNDV_RAILS=1`: Set this to `1` to disable multi-rail transfers in UCX, where UCX splits
+  data to utilize various channels (e.g. two NICs). A value greater than `1` can cause a performance drop, 
+  for high-bandwidth transports between GPUs.
+- `UCX_MEMTYPE_CACHE=n`: Disables a cache in UCX that can cause UCX to fail when running with CUDA buffers. 
+- `UCX_RNDV_SCHEME=put_zcopy`: Picks the scheme to be used in the [RNDV](https://community.mellanox.com/s/article/understanding-tag-matching-for-developers)
```

Comment:
Do we want users setting this?  I thought we were moving away from setting this explicitly.  I also thought we saw good performance when letting this be the default.

Reply:
Removed. I tested some of our queries with `auto` and yes it seems it is the way to go at this stage.